save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RBuilderUtils.kt

```diff
@@ -32,13 +31,15 @@ enum class ConfirmationType {
  * @param isErrorOpen flag to handle error
  * @param errorLabel label of error
  * @param errorMessage message of error
+ * @param closeButtonLabel label that will be shown on the only button
  * @param handler handler to close
  * @return modal
  */
 fun RBuilder.runErrorModal(
```

Comment:
These things can be refactored: converted to a `FC` with props containing all these settings, and renamed from `run...Modal` to something like `singleButtonModal` and `doubleButtonModal`. Can be done as a separate PR